VoiceController.php

```diff
@@ -1,48 +1,53 @@
-public function voice(Request $request){
-    $request->validate([
-        'question_id'=>'required|int|exists:questions,id',
-        'value'=>'required|boolean',
-    ]);
+<?php
+public function voice(VoiceRequest $request){
+    
+    // Moved requset to validation
+    $request->validated();
 
+    // FindOrFail is suitable here? Never used in api
```

Comment:
@debjit it is suitable for APIs, but the message would be general "Model not found" instead of this "Not found question" that the author wants here.